app.py

```diff
@@ -48,6 +48,10 @@ class RequestForm(Form):
             ]
             )
     email_confirm = TextField('Confirm Email')
+    captcha = TextField(
+            'Capital of Germany (to make sure that you are human)',
+            validators=[validators.AnyOf('Berlin', 'berlin')]
```

Comment:
Maybe we need to add a Required here? I can leave this field blank at the moment.